### PR TITLE
dev-util/cutter: Bump to 1.10.1

### PR DESCRIPTION
**Pull request by ngg**
Closes: https://bugs.gentoo.org/708886
Signed-off-by: Gergely Nagy <ngg@ngg.hu>

### CODE REVIEW

File: dev-util/cutter/cutter-1.10.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+DEPEND="
+	${PYTHON_DEPS}
+	>=dev-qt/qtcore-5.9.1:5
+	>=dev-qt/qtgui-5.9.1:5
+	>=dev-qt/qtsvg-5.9.1:5
+	>=dev-qt/qtwidgets-5.9.1:5
+	dev-qt/qtnetwork:5
+	>=dev-util/radare2-4.2.1
+	jupyter? ( dev-python/jupyter )
```
Comment (juippis) [inline dev-util/cutter/cutter-1.10.1.ebuild]: Does this perhaps require the same python interpreter to be used as in this ebuild? Ie, if I have jupyter installed for python3.7, will it work with 3.6 on this one?

Comment (thestr4ng3r) [reply dev-util/cutter/cutter-1.10.1.ebuild]: Jupyter here makes no sense at all, it has been moved to a plugin long ago: https://github.com/radareorg/cutter-jupyter

File: dev-util/cutter/cutter-1.10.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-1.8.0-python3-config.patch"
+)
+
+src_configure() {
+	local myqmakeargs=(
+		CUTTER_ENABLE_JUPYTER=$(usex jupyter true false)
```
Comment (thestr4ng3r) [inline dev-util/cutter/cutter-1.10.1.ebuild]: this option does not exist anymore

File: dev-util/cutter/cutter-1.10.1.ebuild (hunk 1 above)
Comment (ngg) [reply dev-util/cutter/cutter-1.10.1.ebuild]: I removed the jupyter & webengine use flags

**Comment by ngg** [comment]
> Any ideas?
Cutter 1.10.1 is only compatible with Radare 4.2.1 and not with >=4.3.0. Unfortunately there are no newer Cutter releases, so I'll also update the ebuild here to force the older radare version.

**Comment by ngg** [comment]
It was straightforward to cherry-pick the fix for newer radare2 versions from https://github.com/radareorg/cutter/pull/2067/commits/cda21e0cb8feaeb7e61b638c0aa0778b613a1bab, it should work for you now.

**Pull request closed without merging**